presto-mixin/dashboards/presto-coordinator.libsonnet

```diff
@@ -124,52 +125,46 @@ local heapMemoryUsagePanel = {
   pluginVersion: '10.2.0-62263',
 };
 
-local normalQueryOneMinuteCountPanel = {
+local errorFailuresOneMinuteCountPanel(legendMatcher, matcher) = {
   datasource: promDatasource,
   targets: [
     prometheus.target(
-      'com_facebook_presto_execution_QueryManager_CompletedQueries_OneMinute_Count{job=~"$job", presto_cluster=~"$presto_cluster", instance=~"$instance"}',
+      'presto_QueryManager_InternalFailures_OneMinute_Count{' + matcher + ', presto_cluster=~"$presto_cluster"}',
       datasource=promDatasource,
-      legendFormat='{{presto_cluster}} - {{instance}} - completed',
+      legendFormat='' + legendMatcher + ' - internal',
       format='time_series',
     ),
     prometheus.target(
-      'com_facebook_presto_execution_QueryManager_RunningQueries{job=~"$job", presto_cluster=~"$presto_cluster", instance=~"$instance"}',
-      datasource=promDatasource,
-      legendFormat='{{presto_cluster}} - {{instance}} - running',
-    ),
-    prometheus.target(
-      'com_facebook_presto_execution_QueryManager_StartedQueries_OneMinute_Count{job=~"$job", presto_cluster=~"$presto_cluster", instance=~"$instance"}',
+      'presto_QueryManager_UserErrorFailures_OneMinute_Count{' + matcher + ', presto_cluster=~"$presto_cluster"}',
       datasource=promDatasource,
-      legendFormat='{{presto_cluster}} - {{instance}} - started',
+      legendFormat='' + legendMatcher + ' - user',
     ),
   ],
   type: 'timeseries',
-  title: 'Normal query - one minute count',
-  description: 'A count of completed, running, and started queries.',
+  title: 'Error failures - one minute count',
```

Comment:
Can you change this to avoid the double negative please? Just "Failures - one minute count" is fine